### PR TITLE
`chore` Fix all warnings

### PR DESCRIPTION
**Pull request by goergisn**
## Summary
- Increase `file_limit` to 500
- Move some code to new files / functions / extensions
- Resolved ACL warnings
- Shortened names

### CODE REVIEW

File: .swiftlint.yml [hunk 1]
```diff
@@ -19,3 +19,6 @@ line_length:
   ignores_function_declarations: true
   ignores_urls: true
   warning: 140
+
+file_length:
+  warning: 500
```
Comment (descorp) [inline .swiftlint.yml:24]: 😆

File: Adyen/Core/Payment Methods/Abstract/PaymentMethodType.swift [hunk 2]
```diff
@@ -180,6 +180,9 @@ public enum PaymentMethodType: RawRepresentable, Hashable, Codable {
         case let .other(value): return value
         }
     }
+}
+
+extension PaymentMethodType {
```
Comment (descorp) [inline Adyen/Core/Payment Methods/Abstract/PaymentMethodType.swift:185]: What does introduction of extension change? 🤔

**Review by descorp — APPROVED** [review]

File: Adyen/Core/Payment Methods/Abstract/PaymentMethodType.swift (hunk 2 above)
Comment (goergisn) [reply Adyen/Core/Payment Methods/Abstract/PaymentMethodType.swift:185]: Body length

**Comment by goergisn** [comment]
Replacing with https://github.com/Adyen/adyen-ios/pull/1468 which is targeting `develop`

**Pull request closed without merging**